samples/benchmark/bunnymark/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,135 @@
+
+val platform: String by project
+val armArch: String by project
+val iosSigningIdentity: String by project
+val buildType: String? by project
+
+buildscript {
+    repositories {
+        mavenLocal()
+        maven("https://dl.bintray.com/utopia-rise/kotlin-godot")
+        jcenter()
+    }
+    dependencies {
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.61")
+        classpath("org.godotengine.kotlin:godot-gradle-plugin:1.0.1")
+    }
+}
+
+plugins {
+    id("org.jetbrains.kotlin.multiplatform") version ("1.3.61")
+}
+
+apply(plugin = "godot-gradle-plugin")
+
+repositories {
+    mavenLocal()
+    maven("https://dl.bintray.com/utopia-rise/kotlin-godot")
+    jcenter()
```

Comment:
It's missing exclusion for kotlinx coroutines module. Even if we don't use them here I think we should still have same template for our samples so that user don't get lost.